src/dymaptic.GeoBlazor.Core/Objects/AbortManager.cs

```diff
@@ -69,32 +70,47 @@ public async Task<IJSObjectReference> CreateAbortSignal(CancellationToken cancel
     /// </param>
     public async Task DisposeAbortController(CancellationToken cancellationToken)
     {
-        (CancellationTokenRegistration registration, IJSObjectReference abortControllerRef) =
-            _tokensAndControllers[cancellationToken];
+        try
+        {
+            (CancellationTokenRegistration registration, IJSObjectReference abortControllerRef) =
+                _tokensAndControllers[cancellationToken];
 
-        await registration.DisposeAsync();
-        await abortControllerRef.DisposeAsync();
+            await registration.DisposeAsync();
+            await abortControllerRef.DisposeAsync();
 
-        if (_tokensAndControllers.ContainsKey(cancellationToken))
+            if (_tokensAndControllers.ContainsKey(cancellationToken))
+            {
+                _tokensAndControllers.TryRemove(cancellationToken, out _);
+            }
+        }
+        catch(Exception ex)
         {
-            _tokensAndControllers.Remove(cancellationToken);
+            Console.WriteLine(ex);
```

Comment:
Errors during object disposal usually have to do with race conditions of the object(s) already being disposed or set to `null`. In this case, I actually fixed the error I saw by switching to `TryRemove`, but I figured, even if we get an unknown error, we probably don't want to crash/throw. So in those cases, where I don't really expect anything, I tend to default to `Console.WriteLine`. I'm open to other suggestions, but if these are rare, a more detailed console message might not make a lot of difference, since it would be often ignored/missed.